shared/src/iosTest/kotlin/co/touchlab/kmp/BaseModelTest.kt

```diff
@@ -0,0 +1,50 @@
+package co.touchlab.kmp
+
+import co.touchlab.kampstarter.BaseTest
+import co.touchlab.kampstarter.ktor.network
+import co.touchlab.kampstarter.models.BaseModel
+import co.touchlab.stately.freeze
+import co.touchlab.stately.isFrozen
+import kotlinx.coroutines.*
+import kotlin.coroutines.CoroutineContext
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class BaseModelTest : BaseTest() {
+    @Test
+    fun testScopeIsolation() = runTest {
+        lateinit var networkContext: CoroutineContext
+        lateinit var controlContext: CoroutineContext
+        val model = BaseModel()
```

Comment:
The test also works if you use `runTest`'s scope instead of instantiating a new model to get a scope. At that point `BaseModelTest` is a bit of a misnomer.